src/frontends/pytorch/src/op/add.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_add(NodeContext& context) {
+    auto rhs = context.get_input(1);
```

Comment:
this is not bad to give some description to the operation being translated because `add` means different from what is implemented:)

Reply:
Same